opensrp-opd/src/main/java/org/smartregister/opd/utils/OpdJsonFormUtils.java

```diff
@@ -407,8 +407,12 @@ public static void saveImage(@NonNull String providerId, @NonNull String entityI
             return;
         }
 
-        Bitmap compressedImageFile = OpdLibrary.getInstance().getCompressor().compressToBitmap(file);
-        saveStaticImageToDisk(compressedImageFile, providerId, entityId);
+        try {
+            Bitmap compressedImageFile = OpdLibrary.getInstance().getCompressor().compressToBitmap(file);
+            saveStaticImageToDisk(compressedImageFile, providerId, entityId);
+        } catch (Exception e) {
+            Timber.e(e);
```

Comment:
1. Is this supposed to be a general exception?
2. Are we supposed to handle this exception if the image was not saved?

Reply:
1. If i put the `IOException` the task to run JacocoTestReport fails
2. Yeah, i think the rest of the information in this form are important if we lose the image because of `IOException`, we should just continue. What do you think?